bin/run_prod_database_migration.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [ -f ./.env ]; then
+  . ./.env
+else
+  exit 1 ".env does not exist"
+fi
+
+wget https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64 -O cloud_sql_proxy
+chmod +x cloud_sql_proxy
+./cloud_sql_proxy -instances=$INSTANCE_CONNECTION_NAME=tcp:5432 &
+
+sleep 4s
```

Comment:
What is this sleep waiting for? Is it potentially flaky?

Reply:
It's not flaky, we don't really need this; but the logs get a bit confusing if not because the proxy isn't finished setting up the connection when you try to connect to it (it blocks so it works).